support-workers/src/main/scala/com/gu/acquisitions/AcquisitionServiceBuilder.scala

```diff
@@ -7,9 +7,9 @@ import com.gu.config.Configuration
 import com.gu.okhttp.RequestRunners
 import com.gu.support.config.Stages
 
-object AcquisitionService {
+object AcquisitionServiceBuilder {
```

Comment:
Having this object called `AcquisitionService` was confusing because it actually constructs a `com.gu.acquisition.services.AcquisitionService`. I think this name better describes its actual purpose

Reply:
scala convention is the apply method in the companion object is the constructor.
However I would agree if it's more than just a constructor (i.e. it's doing some decision making) then it makes sense to be something in its own right.
The latter appears to be the case here by the looks, so your change makes sense! 😄